tests/test_auxtel_disable_ataos_corrections.py

```diff
@@ -40,24 +41,24 @@ async def basic_make_script(self, index):
         self.script = DisableATAOSCorrections(index=index)
         self.atcs_mock = ATCSMock()
 
-        return (self.script, self.atcs_mock)
+        self.script.atcs = ATCS(
```

Comment:
These unit tests were doing an actual test of the execution, see that they are using the `ATCSMock` above. Any reason for changing them? If not, I would rather leave them as they were.